### DISCUSSION TITLE
Where do you report an issue with CAP ?

### DISCUSSION THREAD

**Discussion by aspirio187** [cap-js/.github#9] [Q&A]
Hi everyone,

I opened an issue on SAP JavaScript Cloud SDK repo but they redirected me to CAP directly, the problem is that the bug or the "situation" is related to destinations and I can't find any repo linked to that.

Do you know where I can post that ?

This is my original post if you need more information on the issue: [Impossible to pass custom headers in BTP Destination _#5061_
](https://github.com/SAP/cloud-sdk-js/issues/5061)

**Comment by gregorwolf** [comment]
Hi @aspirio187 ,

for such an issue you should use the [Support Channels](https://cap.cloud.sap/docs/resources/#support-channels) via https://me.sap.com/servicessupport.

Best Regards
Gregor
